modules/meeting/db/migrate/20251022114752_add_participation_status_for_meeting_participant.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddParticipationStatusForMeetingParticipant < ActiveRecord::Migration[8.0]
+  def change
+    add_column :meeting_participants, :participation_status, :string, null: true
```

Comment:
@klaustopher why not a native Postgres enum? I know there are issues with deleting values from native `ENUM` types, but I would argue that this is unlikely to happen very often this case since our values map to a defined industry standard (iCalendar).

Reply:
There have been issues with enums and multiple schemas in the past, so I think having a string enum is the safer option